tests/integration/test_events.py

```diff
@@ -367,6 +367,7 @@ def _put_events_with_filter_to_sqs(
         sqs_client.set_queue_attributes(
             QueueUrl=queue_url, Attributes={"Policy": json.dumps(policy)}
         )
+        # ALLOWHERE
```

Comment:
What is this supposed to mean?

Reply:
That is a very good question, which I am currently unable to answer? I guess I wanted to use this method at first, but realised it didn't fit my need, took some note? Maybe I wanted to replace this with `events_allow_event_rule_to_sqs_queue` fixture, I think? Anyway I delete that comment 😂 thanks!